src/debugger/nodeDebugWrapper.ts

```diff
@@ -146,7 +147,7 @@ export class NodeDebugWrapper {
      * - Create global logger
      */
     private requestSetup(debugSession: NodeDebugSession, args: any) {
-        this.projectRootPath = path.resolve(args.program, "../..");
+        this.setProjectRoot(args);

```

Comment:
Nit: If the only purpose of `setProjectRoot` is to set `this.projectRootPath`, IMO it'd be more transparent and readable to make a function that would return `projectRootPath`, and use it's result.